core/node/options.go

```diff
@@ -138,7 +140,10 @@ func GetOptions() *Options {
 
 // GetLogOptions retrieves logger options from the app configuration.
 func GetLogOptions() *logconfig.LogOptions {
-	logDir := config.GetString(config.FlagLogDir)
+	filepath := ""
+	if logDir := config.GetString(config.FlagLogDir); logDir != "" {
```

Comment:
Not sure if we need to append with custom path if user sets his own logDir. I would expect logs to be written to my specified directory.

Reply:
This builds the filepath using given log directory. Filepath is directory + filename.